packages/xo-web/src/common/icon.js

```diff
@@ -23,3 +23,20 @@ Icon.propTypes = {
 }
 
 export { Icon as default }
+
+export const StackedIcons = ({ icons, ...props }) => (
+  <span className='fa-stack'>
+    {icons.map(({ icon, size }) => (
+      <Icon key={icon} icon={icon} size={`stack-${size}x`} {...props} />
```

Comment:
We should be able to customize a specific icon.

Reply:
It's a generic component we can limit it to two props or we can let it flexible by making it configurable. (for example, if we need to change a color or size of a specific icon).

I did this comment because i see that the rest of `StackedIcons` props is affected to all icon children which will limit the implementation when will need to add some spaces to the global component.